bot/cogs/moderation/moderation_commands.py

```
@@ -0,0 +1,76 @@
+    @commands.hybrid_command(name="ban", description="Ban a user")
+    @commands.has_permissions(manage_messages=True)
+    @guard("ban")
+    async def ban(self, ctx: commands.Context, member: discord.Member):
+        # Check Moderation Settings
+        settings = await self.get_mod_settings(str(ctx.guild.id))
+        if not settings.is_moderation_settings_enabled:
+            await ctx.send("⚠️ Moderation system is disabled in this server.")
+            return
+
+        if not member:
+            await ctx.send(f"If you are not trying to ban a ghost, please mention a member.")
```

Ruff (0.14.6)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)